Makefile

```diff
@@ -48,3 +50,7 @@ test-assets:
 	python pytestgeventwrapper.py rotkehlchen/tests/exchanges/test_independentreserve.py::test_assets_are_known
 	python pytestgeventwrapper.py rotkehlchen/tests/unit/test_zerionsdk.py::test_protocol_names_are_known
 	python pytestgeventwrapper.py rotkehlchen/tests/unit/test_zerionsdk.py::test_query_all_protocol_balances_for_account
+
+
+create-cassettes:
```

Comment:
I think we need to do it per test. Not for all. I would never run it for all locally and running it for all increases the chance it will record backoffs. And we need to do it locally ofc.